configurator/config.json

```diff
@@ -33,14 +33,18 @@
     "enforce_basepath": false,
     "ignore_pattern": [
       "__pycache__"
-    ]
+    ],
+    "deployment_key": [],
+    "deployment_key_protocol": "rsa"
   },
   "schema": {
     "dirsfirst": "bool",
     "enforce_basepath": "bool",
     "ignore_pattern": [
       "str"
-    ]
+    ],
+    "deployment_key": ["str"],
+    "deployment_key_protocol": "match(rsa|dsa|ecdsa|ed25519|rsa)"
```

Comment:
Been thinking about this line... because it made no sense to me why it is even there...

Figured, because this PR is writing the deployment key to disk.

Just a thought, wouldn't it make more sense to start the ssh-agent and just add the key in on the fly?

Reply:
Sounds good to me - I just copied the approach (and code!) of the git_pull add-on. I'm completely new to Docker and not so hot on bash. I'll try using `ssh-add` instead, as suggested.